torch/csrc/jit/mobile/upgrader_mobile.cpp

```diff
@@ -1,257 +1,302 @@
-#include <caffe2/serialize/versions.h>
+/**
+ * @generated
+ * This is an auto-generated file. Please do not modify it by hand.
+ * To re-generate, please run:
+ * cd ~/pytorch && python torch/csrc/jit/mobile/upgrader_mobile.cpp
+ */
+
 #include <torch/csrc/jit/mobile/upgrader_mobile.h>
 
-namespace c10 {
-TypePtr parseType(const std::string& pythonStr);
-} // namespace c10
```

Comment:
Looks like these lines were missing in the codegen, could you help with adding them?

Reply:
I meant adding them as part of `gen_mobile_upgraders.py`